test/cpp/end2end/xds_end2end_test.cc

```diff
@@ -2290,6 +2290,7 @@ TEST_P(XdsResolverOnlyTest, CircuitBreaking) {
     Status status_;
   };
 
+  gpr_setenv("GRPC_ENABLE_CIRCUIT_BREAKING", "true");
```

Comment:
Please also add a test that the circuit breaking config is *not* honored when this env var is not set.